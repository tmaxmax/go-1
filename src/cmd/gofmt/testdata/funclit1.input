//gofmt -s -R

package funclit1

func _() {
    f = func(a, b int) int { return a + b }
    f = func(a, b int) (int, string) {
        return a + b, strconv.Itoa(a * b)
    }
	f = func() (_, _, _, _, _) {
		return "long", "expression", "list", "this", "should absolutely be printed with return on a separate line because it is too long"
	}
}
