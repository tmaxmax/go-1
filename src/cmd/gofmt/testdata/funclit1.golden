//gofmt -s -R

package funclit1

func _() {
    f = func { a, b | a + b }
    f = func { a, b | a + b, strconv.Itoa(a * b) }
	f = func {
		return "long", "expression", "list", "this", "should absolutely be printed with return on a separate line because it is too long"
	}
}
